src/helpers/request-body.js

```
@@ -0,0 +1,49 @@
+        body: JSON.stringify(body),
+        contentType: normalizedContentType || 'application/json'
+      };
+    }
+
+    return { body, contentType: normalizedContentType };
+  }
+
+  window.CrossRequestHelpers = window.CrossRequestHelpers || {};
+  window.CrossRequestHelpers.serializeRequestBody = serializeRequestBody;
+
+  if (typeof module !== 'undefined' && module.exports) {
```

ESLint

[error] 'module' is not defined.

(no-undef)